azurerm/internal/services/cosmos/cosmosdb_account_resource_test.go

```diff
@@ -993,3 +1016,60 @@ func checkAccAzureRMCosmosDBAccount_basic(data acceptance.TestData, consistency
 		resource.TestCheckResourceAttrSet(data.ResourceName, "secondary_readonly_key"),
 	)
 }
+
```

Comment:
@katbyte @jackofallops I've created a test using a key vault example from a different test, can I get some 👀 on it? And run it as an acc-test if it looks correct?